docs/user_guide.md

```diff
@@ -258,18 +258,16 @@ kubectl describe flinkclusters <CLUSTER-NAME>
 
 ### Flink job
 
-To get a list of jobs
+In a job cluster, the job is automatically submitted by the operator.
+The operator creates a submitter for a Flink job.
```

Comment:
Add "The submitter itself is a Kubernetes job".

Reply:
Updated.